.github/workflows/docker-image.yml

```diff
@@ -115,8 +126,10 @@ jobs:
           sudo rm -rf /usr/local/lib/android
           sudo rm -rf /opt/hostedtoolcache/CodeQL
           
-          if [[ "${image_type}" == "gravitino" || "${image_type}" == "iceberg-rest-server" ]]; then
+          if [[ "${publish_latest}" == "true" ]]; then
+            echo "Publish latest tag"
             ./dev/docker/build-docker.sh --platform all --type ${image_type} --image ${image_name} --tag ${{ github.event.inputs.version }} --latest
           else
+            echo "Doesn't publish latest tag"
             ./dev/docker/build-docker.sh --platform all --type ${image_type} --image ${image_name} --tag "${tag_name}-${{ github.event.inputs.version }}"
```

Comment:
This `if...else...` is not correct, we should always build the image with tag names, right?